sequencer/src/bin/count-transactions.rs

```diff
@@ -79,8 +79,8 @@ async fn main() {
                         }
                     }
                 };
-                for i in 0..block.len() {
-                    let txn = block.transaction(&(i as u64)).unwrap();
+                for t in block.payload().iter(block.payload().get_ns_table()) {
+                    let txn = block.transaction(&t).unwrap();
```

Comment:
Shorthand for this: `for (t, txn) in block.enumerate() {`

Reply:
That's much cleaner but still not squeaky clean 3347b8aef8b2a98121d8bc847c000bef82e87fd8.  I had to replace `.hash()` with `.commit()` and it looks like
```rust
                for (_, txn) in block.enumerate() {
                    total += 1;
                    unique.insert(txn.commit());
                }
```